### ISSUE TITLE
How to predict a Single Image after Model is trained?

### ISSUE THREAD

**Issue by jaytimbadia** [phlippe/uvadlc_notebooks#16]
Hey,

Wonderful Repo!

I have trained a protoMAML network from scratch on food dateset. Model is trained. 
SO now if I want to predict a single Image, so that to ask it falls into which category, there is no way described in the notebook to do that?

Can you please let me know how can I do that? 
In short, I have trained on 81 classes of food and now I want to pass an Image and see how model gives confidence accuracy (TOP -5).

Please Help!

**Comment by phlippe** [comment]
Hi, a trained ProtoMAML model is different from standard classification models by the fact that it represents rather a parameter initialization than a final model. In other words, ProtoMAML and any other optimization-based meta-learning method is designed to be finetuned on a small test set / support set before evaluating its classification performance. In your case, you can take the trained ProtoMAML model and finetune it on a batch of all 81 classes, similar to what is done for the test set in the `test_protomaml` function. After that, you use the finetuned model to predict the classification output on your new image.

**Comment by jaytimbadia** [comment]
Great. Thank you so much! Let me try that and come back.
